localnet/poktrolld/config/supplier1_stake_config.yaml

```diff
@@ -2,17 +2,17 @@ owner_address: pokt1mrqt5f7qh8uxs27cjm9t7v9e74a9vvdnq5jva4
 operator_address: pokt1mrqt5f7qh8uxs27cjm9t7v9e74a9vvdnq5jva4
 stake_amount: 1000069upokt
 default_rev_share_percent:
-  - pokt1mrqt5f7qh8uxs27cjm9t7v9e74a9vvdnq5jva4: 80.5
-  - pokt1eeeksh2tvkh7wzmfrljnhw4wrhs55lcuvmekkw: 19.5
+  pokt1mrqt5f7qh8uxs27cjm9t7v9e74a9vvdnq5jva4: 80.5
+  pokt1eeeksh2tvkh7wzmfrljnhw4wrhs55lcuvmekkw: 19.5
 services:
   - service_id: anvil
     endpoints:
       - publicly_exposed_url: http://relayminer1:8545
         rpc_type: JSON_RPC
   - service_id: ollama
     rev_share_percent:
-      - pokt1mrqt5f7qh8uxs27cjm9t7v9e74a9vvdnq5jva4: 50
-      - pokt1eeeksh2tvkh7wzmfrljnhw4wrhs55lcuvmekkw: 50
```

Comment:
Thanks for the fix. I wonder how this did work in the past :thinking:

Reply:
It did not work :sweat_smile:

Reply:
How did you test it?

Reply:
These specific config files are not being used. E2E tests are generating their own.